MaterialLib/MPL/CheckMaterialSpatialDistributionMap.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "MaterialSpatialDistributionMap.h"
+#include "Medium.h"
+#include "MeshLib/Mesh.h"
+#include "Phase.h"
+
+namespace MaterialPropertyLib
+{
+template <typename ContainerMedium,
+          typename ContainerSolid,
+          typename ContainerLiquid>
+void checkMaterialSpatialDistributionMap(
+    MeshLib::Mesh const& mesh,
+    MaterialPropertyLib::MaterialSpatialDistributionMap const& media_map,
+    ContainerMedium const& required_properties_medium,
+    ContainerSolid const& required_properties_solid_phase,
+    ContainerLiquid const& required_properties_liquid_phase)
```

Comment:
The one for gas (or no-wetting) phase is missing.

Reply:
Yes, you are right. It is planned in another PR.

Reply:
I see